src/App.tsx

```diff
@@ -1,14 +1,21 @@
 import React from 'react';
-
+import { PostList } from './components/PostList/PostList';
 import './App.scss';
+import posts from './api/posts';
+import comments from './api/comments';
+import users from './api/users';
+import { Post } from './types/Post';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+const preparedPosts: Post[] = posts.map(post => ({
+  ...post,
+  user: users.find(user => user.id === post.id) || null,
+  comments: comments.filter(comment => comment.postId === post.id) || [],
```

Comment:
Unnecessary `||`, `filter` will return an empty array if no elements would pass the check
```suggestion
  comments: comments.filter(comment => comment.postId === post.id),
```